### ISSUE TITLE
Add roles added in ARIA 1.2

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#2107]
WAi-ARIA 1.2 has added a few new roles. Those should be added to axe-core. These roles aren't likely to effect accessibility when they're not supported so I think it's safe to add these right away.

**Comment by straker** [comment]
See [the ariaRole standards docs](https://github.com/dequelabs/axe-core/blob/develop/doc/standards-object.md) for how to do this

**Comment by straker** [comment]
New roles for ARIA 1.2

* [blockquote](https://www.w3.org/TR/wai-aria-1.2/#blockquote)
* [caption](https://www.w3.org/TR/wai-aria-1.2/#caption)
* [code](https://www.w3.org/TR/wai-aria-1.2/#code)
* [deletion](https://www.w3.org/TR/wai-aria-1.2/#deletion)
* [emphasis](https://www.w3.org/TR/wai-aria-1.2/#emphasis)
* [insertion](https://www.w3.org/TR/wai-aria-1.2/#insertion)
* [meter](https://www.w3.org/TR/wai-aria-1.2/#meter)
* [paragraph](https://www.w3.org/TR/wai-aria-1.2/#paragraph)
* [strong](https://www.w3.org/TR/wai-aria-1.2/#strong)
* [subscript](https://www.w3.org/TR/wai-aria-1.2/#subscript)
* [superscript](https://www.w3.org/TR/wai-aria-1.2/#superscript)
* [time](https://www.w3.org/TR/wai-aria-1.2/#time)

**Comment by timmybytes** [comment]
Hey there! I'm trying to gain more experience with accessibility, and I'd love to work on this issue!

**Comment by straker** [comment]
All yours. Let me know if you need anything.

**Comment by straker** [comment]
PS. I find running the following script while the WCAG 1.2 spec is open helps seeing which ARIA attributes to add to the `requiredAttrs` and `allowedAttrs` properties by hiding any global ones:

```js
const globalAttrs = Array.from(
    document.querySelectorAll('#global_states li')
  ).map(li => li.textContent.replace(/\s*\(.*\)/, ''));

  const globalRoleAttrs = Array.from(
    document.querySelectorAll('.role-inherited li, .role-properties li')
  ).filter(li => globalAttrs.includes(li.textContent.replace(/\s*\(.*\)/, '')))

  globalRoleAttrs.forEach(li => li.style.display = 'none');
```

**Comment by straker** [comment]
dev task
